content/en-us/reference/engine/classes/Explosion.yaml

```diff
@@ -15,9 +15,7 @@ description: |
   `Class.Humanoid` characters not protected by a `Class.ForceField`.
   `Class.Constraint|Constraints` will not be broken by an explosion.
 
-  If an explosion is instanced while the game is running, it will destroy itself
-  shortly afterwards, so they do not need to be cleaned up using the
-  `Class.Debris` service.
+  If an explosion is parented while the game is running, it will automatically play the explosion effects and unparent itself when it finishes. Note that it is not destroyed, so connections do not get disconnected and the parent is not locked. If you hold open a connection containing a strong reference to the explosion, it will not be garbage collected.
```

Comment:
```suggestion
  If an explosion is parented while the experience is running, it will automatically play the explosion effects and unparent itself when it finishes. It is not destroyed in this case, so connections do not get disconnected and the parent is not locked. If you hold open a connection containing a strong reference to the explosion, it will not be garbage collected.
```

Reply:
The last sentence, does that imply something even more specific to Explosion than typically holding a strong reference to a connection that would prevent garbage collection?

Reply:
Also, in the first sentence, does it matter where it's parented to? If so, perhaps we should be more specific...

Reply:
Clarified the parent can be anywhere in the data model, and that garbage collection is not unique to this instance. I think it's still valuable to include a reminder here to give further meaning to why the distinction that it's not destroyed is important, given that this is a common pitfall for memory leaks.